### DISCUSSION TITLE
android box

### DISCUSSION THREAD

**Discussion by rathorp80** [OMOTE-Community/OMOTE-Firmware#25] [General]
how hard would it be to change the apple tv menu to a android box

**Comment by CoretechR** [comment]
You would just have to swap the icons and the IR codes for the buttons. The Android box can be controlled via infrared, right?

**Comment by rathorp80** [comment]
yes, is there a video where i can learn to do this??

**Reply by CoretechR** [reply]
At the moment you will have to edit the code directly to customize the UI. First, you need to figure out the IR code for your Android box and change the main.cpp to use these codes for the "Apple TV" menu. Then you can use the [LVGL converter](https://lvgl.io/tools/imageconverter) convert your logo so it can be placed inside the assets file. 
I can try to guide you through this, but there is no tutorial for this yet.

**Reply by rathorp80** [reply]
i would appreciate that at the moment im still waiting for all my parts to arrive
